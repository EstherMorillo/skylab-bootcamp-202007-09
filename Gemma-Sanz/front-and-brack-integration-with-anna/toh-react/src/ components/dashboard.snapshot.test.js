import renderer from 'react-test-renderer';
import React from 'react';
import Dashboard from './dashboard';
import { BrowserRouter as Router } from 'react-router-dom';

describe('App snapshot', () => {
	const tree = renderer.create(
		<Router>
			<Dashboard />
		</Router>
	);

	it('should match', () => {
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
