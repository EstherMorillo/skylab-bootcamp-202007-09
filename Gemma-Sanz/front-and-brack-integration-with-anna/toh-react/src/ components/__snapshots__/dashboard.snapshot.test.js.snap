// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
<div>
  <h2>
    Top four
  </h2>
  <ul
    className="dashboard_s_list row"
  />
</div>
`;
